### DISCUSSION TITLE
credentiel params

### DISCUSSION THREAD

**Discussion by mhammouda86** [aws/aws-sdk-php#2717] [Q&A] [answered]
Hello,
How i can set credentiel params for developping?
`$client = new SecretsManagerClient([
    'profile' => 'default',
    'version' => '2017-10-17',
    'region' => 'us-west-2',
    'credentials' => [
        'key' => $key,
        'secret' => $secret,
    ]
]);`

How i can set The $key and $secret  or find it?

Thanks

**Comment by yenfryherrerafeliz** [comment, marked as answer]
Hi @mhammouda86, here is a quick example of how to provide credentials when initializing a client:

```php
<?php

use Aws\SecretsManager\SecretsManagerClient;

$client = new SecretsManagerClient([
    'version' => 'latest',
    'region' => 'REGION',
    'credentials' => [
        'accessKeyId' => 'YOUR_ACCESS_KEY',
        'secretAccessKey' => 'YOUR_SECRET_ACCESS_KEY'
    ]
]);

```

I hope this help!

Thanks!

**Reply by mhammouda86** [reply]
Have method to generate dev access key and secret?

[a comment by an automated account is not shown]
